classes/class-wc-connect-shipping-method.php

```diff
@@ -60,15 +60,19 @@ public function __construct( $id_or_instance_id = null ) {
 				$this->title = '';
 			} else {
 				$this->id = $this->service_schema->method_id;
-				$this->method_title = $this->service_schema->method_title;
+				// If there is already a shipping method with the "$service_schema->id" ID ("usps", "canada_post", etc),
+				// use the longer "method_title" property for the method title, which has the "XXX (WooCommerce Services)" disambiguation
+				$this->method_title = isset( $existing_shipping_methods[ $this->service_schema->id ] )
+					? $this->service_schema->method_title
+					: $this->service_schema->carrier_name;
 				$this->method_description = $this->service_schema->method_description;
 				$this->supports = array(
 					'shipping-zones',
 					'instance-settings'
 				);
 
 				// Set title to default value
-				$this->title = $this->service_schema->method_title;
+				$this->title = $this->service_schema->carrier_name;
```

Comment:
If another plugin is added after a WCS shipping method is added, then the title will still show the carrier name instead of the method title, for example with `USPS`:

<img width="935" alt="screen shot 2018-08-13 at 4 44 37 pm" src="https://user-images.githubusercontent.com/11487924/44057319-c88129f4-9f18-11e8-802a-19831db96c33.png">
<img width="709" alt="screen shot 2018-08-13 at 4 44 31 pm" src="https://user-images.githubusercontent.com/11487924/44057323-caea0706-9f18-11e8-8ca2-f92e034526a8.png">

Those are WCS shipping methods, and can be confusing if USPS is installed later.

Probably not a huge issue, and would require a lot of work to fix (dynamically getting the method title when displaying), so ok without fixing it for now.